File: retail industries Suite -all/home_makeover_pro/home_makeover_pro/home_makeover_management/report/project_status_dashboard/project_status_dashboard.py
import frappe


def execute(filters=None):
    columns = [
        {"label": "Project", "fieldname": "name", "fieldtype": "Link",
         "options": "Home Makeover Project", "width": 150},
        {"label": "Customer", "fieldname": "customer", "fieldtype": "Link",
         "options": "Customer", "width": 150},
        {"label": "Type", "fieldname": "project_type", "fieldtype": "Data", "width": 110},
        {"label": "Status", "fieldname": "workflow_state", "fieldtype": "Data",
         "width": 110},
        {"label": "Budget Estimate", "fieldname": "budget_estimate",
         "fieldtype": "Currency", "width": 130},
        {"label": "Expected End", "fieldname": "expected_end_date",
         "fieldtype": "Date", "width": 110},
    ]
    data = frappe.db.sql("""
        SELECT name, customer, project_type, workflow_state,
               budget_estimate, expected_end_date
        FROM `tabHome Makeover Project`
        ORDER BY creation DESC
    """, as_dict=1)
    return columns, data

File: retail_suite/retail_suite/home_makeover_management/report/project_status_dashboard/project_status_dashboard.js
// Copyright (c) 2026, Your Organization and contributors
// For license information, please see license.txt

frappe.query_reports["Project Status Dashboard"] = {
	filters: [
		{
			fieldname: "workflow_state",
			label: __("Status"),
			fieldtype: "Select",
			options: "\nLead\nConsultation\nDesign\nApproved\nExecution\nCompleted\nCancelled",
			default: "",
		},
		{
			fieldname: "customer",
			label: __("Customer"),
			fieldtype: "Link",
			options: "Customer",
			default: "",
		},
		{
			fieldname: "project_type",
			label: __("Project Type"),
			fieldtype: "Select",
			options: "\nFull Home\nKitchen\nLiving Room\nBedroom\nOffice",
			default: "",
		},
	],
};
